LuaUI/Widgets/gui_notAchili_widgetselector.lua

```diff
@@ -0,0 +1,380 @@
+function widget:GetInfo()
```

Comment:
Last time it was broken... Now is OK, right?

Reply:
I'm not sure what last time you mean (the first versions two weeks ago? - if you mean that, it was addressed in [b3e92fb])

Reply:
I pressume you fixed that then... Anyway, during the tests I'll probably check

Reply:
Checked!